tests/unit/processor/pseudonymizer/test_pseudonymizer.py

```diff
@@ -73,180 +784,71 @@ def test_config_validation(self, config_change, error, msg):
+            self.regex_mapping = regex_mapping
+        self._load_specific_rule(rule)
         self.object.process(event)
-        pseudonyms = self.object.pseudonyms
-
-        assert (
-            event["something"]
-            == "<pseudonym:8d7e9ea64b00d7df5dd7d4e1c9dde8a0b70815eea27bddb67738502f4ea0d2ee>"
-        )
-        assert len(pseudonyms) == 1 and set(pseudonyms[0]) == {"pseudonym", "origin"}
+        assert event == expected, testcase
 
-    def test_init_tld_extractor_uses_file(self):
+    def test_tld_extractor_uses_file(self):
         config = deepcopy(self.CONFIG)
         config["tld_lists"] = [TLD_LIST]
         object_with_tld_list = Factory.create({"pseudonymizer": config}, self.logger)
-        object_with_tld_list._init_tld_extractor()
         assert len(object_with_tld_list._tld_extractor.suffix_list_urls) == 1
         assert object_with_tld_list._tld_extractor.suffix_list_urls[0].endswith(
             "tests/testdata/mock_external/tld_list.dat",
         )
 
-    def test_recently_stored_pseudonyms_are_not_stored_again(self):
-        self.object._cache._max_timedelta = CACHE_MAX_TIMEDELTA
-        event = {"event_id": 1234, "something": "something"}
-
-        rule_dict = {
-            "filter": "event_id: 1234",
-            "pseudonymizer": {"pseudonyms": {"something": "RE_WHOLE_FIELD"}},
-            "description": "description content irrelevant for these tests",
-        }
-
-        self._load_specific_rule(rule_dict)
-        for index in range(3):
-            copied_event = deepcopy(event)
-            self.object.process(copied_event)
-            pseudonyms = self.object.pseudonyms
-            assert (
-                copied_event["something"]
-                == "<pseudonym:8d7e9ea64b00d7df5dd7d4e1c9dde8a0b70815eea27bddb67738502f4ea0d2ee>"
-            )
-            assert len(pseudonyms) == 1, f"step {index}"
-
-            copied_event = deepcopy(event)
-            self.object.process(copied_event)
-            pseudonyms = self.object.pseudonyms
-            assert (
-                copied_event["something"]
-                == "<pseudonym:8d7e9ea64b00d7df5dd7d4e1c9dde8a0b70815eea27bddb67738502f4ea0d2ee>"
-            )
-            assert len(pseudonyms) == 0
-
-            time.sleep(CACHE_MAX_TIMEDELTA.total_seconds())
-
     def _load_specific_rule(self, rule):
-        self.object._load_regex_mapping(self.regex_mapping)
+        self.object._config.regex_mapping = self.regex_mapping
+        del self.object.__dict__["_regex_mapping"]
         super()._load_specific_rule(rule)
         self.object._replace_regex_keywords_by_regex_expression()
 
-    def test_pseudonymization_of_field_fails_because_filter_does_not_match(self):
-        event = {"event_id": 1105, "something": "Not pseudonymized"}
+    def test_pseudonymize_url_fields_not_in_pseudonymize(self):
+        pseudonym = "<pseudonym:d95ac3629be3245d3f5e836c059516ad04081d513d2888f546b783d178b02e5a>"
```

Comment:
It's a bit difficult to know where this pseudonym is coming from. I assume you precomputed it. Could you either add the computation to the test as well or add a comment what the raw value to the pseudonym is? That way I can easier understand what is actually being tested here.

Reply:
ah nevermind, it's the pseudonym of "test" right?